### PR TITLE
Add expiration threshold environment variable for app config ios and …

### PR DESCRIPTION
**Pull request by ioangut**
## Checklist

* [x] Test your changes as thoroughly as possible before you commit them. Preferably, automate your test by unit/integration tests.
* [x] Make sure `mvn -P integration-test clean verify` runs for the whole project and, if you touched any code for a service in the `services` folder, ensure it can be run with `spring-boot:run`

## Description

Add expiration threshold in days environment variable for app config ios and android.

### CODE REVIEW

File: services/distribution/src/main/java/app/coronawarn/server/services/distribution/config/DistributionServiceConfig.java [hunk 1]
```diff
@@ -1603,6 +1603,8 @@ public void setRequireEvaluationTypeHardwareBacked(Boolean requireEvaluationType
 
       private DgcTestCertificateParameters dgcTestCertificateParameters;
 
+      private Integer expirationThresholdInDays;
```
Comment (hilmarf) [inline services/distribution/src/main/java/app/coronawarn/server/services/distribution/config/DistributionServiceConfig.java:1606]: @mlenkeit - should we define some `@Min(0)` or `@Max(100)` values?

**Review by hilmarf — APPROVED** [review]

File: services/distribution/src/main/java/app/coronawarn/server/services/distribution/config/DistributionServiceConfig.java (hunk 1 above)
Comment (mlenkeit) [reply services/distribution/src/main/java/app/coronawarn/server/services/distribution/config/DistributionServiceConfig.java:1607]: @hilmarf  Good idea!

Comment (hilmarf) [reply services/distribution/src/main/java/app/coronawarn/server/services/distribution/config/DistributionServiceConfig.java:1607]: which values make most sense?

File: services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/appconfig/ApplicationConfigurationV2PublicationConfig.java [hunk 2]
```diff
@@ -534,6 +534,8 @@ private DGCParameters buildDgcParameters(
         .build();
     return DGCParameters.newBuilder()
         .setTestCertificateParameters(testCertificateParameters)
+        .setExpirationThresholdInDays(distributionServiceConfig.getAppConfigParameters()
+            .getDgcParameters().getExpirationThresholdInDays())
```
Comment (FelixRottler) [inline services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/appconfig/ApplicationConfigurationV2PublicationConfig.java:538]: Is it possible that ExpirationThresholdInDays will be null? I remember that for some other properties we have a null check ( at least for local stats)

**Review by FelixRottler — APPROVED** [review]

**Pull request merged**